examples/vue-example/src/App.vue

```diff
@@ -872,8 +766,8 @@ onUpdated(() => {
     useWalletKey: useWalletKey.value,
     selectedBuildEnv: selectedBuildEnv.value,
     emailFlowType: emailFlowType.value,
-    enableEd25519Key: enableEd25519Key.value,
-    isEnableMFA: isEnableMFA.value,
+    selectedMFALevel: selectedMFALevel.value,
+    selectedCurveType: selectedCurveType.value,
     customSdkUrl: customSdkUrl.value,
```

Comment:
should also store and rehydrate whitelabel settings

Reply:
Updated by store mfaLevel, curve under whitelabel